### DISCUSSION TITLE
[Feature] Log in with domain name instead of local address.

### DISCUSSION THREAD

**Discussion by smileyface24** [immich-app/immich#8230] [Feature Request]
### The feature

Hello there!

Please kindly consider in the near future Log in with domain name instead of local address.

for example... actually in mobile for login you need enter a local address would be fantastic if you can enter your own domain for login like mydomain.com 

That would be awesome for everyone. 

Thanks. 

Regards.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by alextran1502** [comment]
You can already do that
